app/services/note_creation_service.rb

```diff
@@ -0,0 +1,18 @@
+class NoteCreationService
```

Comment:
This class is kind of unnecessary right now. A simple `Note.create(..)` in the needed place is enough